.github/workflows/java_jni.yml

```
@@ -51,7 +51,7 @@ jobs:
     name: AMD64 manylinux2014 Java JNI
     runs-on: ubuntu-latest
     if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
-    timeout-minutes: 90
+    timeout-minutes: 300
```

Can we revert this before we merge this?

---

I am curious on what will happen once is merged as we don't push the new docker image on non-main branches.
Probably we will have to merge a temporary increase and then decrease but we can see once is merged
```suggestion
    timeout-minutes: 90
```